src/modules/transaction/configuration/moduleAssets.js

```diff
@@ -1,11 +1,13 @@
+import { cryptography } from '@liskhq/lisk-client';
+
```

Comment:
@ManuGowda @eniolam1000752 This blank line can be removed

Reply:
The blank like required to separate import and rest of the code?